hash/m20.xml

```diff
@@ -470,4 +475,630 @@ license:CC0-1.0
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="mazedaze.img" size="286720" crc="dc2bc17c" sha1="dcb79e30338909532bb906c05bca89b1105ea1d9"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="micromissiles">
+        <description>Ballonschießen</description>
+        <year>1982</year>
+        <publisher>C.Marshall, D.T.Laycock</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="micromissiles.img" size="286720" crc="352c67e1" sha1="1f3bf6c9e4832add0d9b882f07fb69ab4974454a"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="mondlandung">
+        <description>Mondlandung</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="mondlandung.img" size="286720" crc="85fbd90a" sha1="7eab04b5a145087cd7d4a67a04285d529e9321a8"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="morse">
+        <description>Morse Code Conversion</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="morse.img" size="286720" crc="2699acd1" sha1="1606eb3a08856615c9bf25464c8bb507dfff8cb1"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="olioids">
+        <description>Astro-Boulders (Oli-oids?) v0.3</description>
+        <year>198?</year>
+        <publisher>Dan W. Altman</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="olioids.img" size="286720" crc="a07a42ae" sha1="5d4ef534694cd51ac5767624c947f4626df89a6e"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="othello_en">
+        <description>Othello (EN)</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="othello_en.img" size="286720" crc="b8b6f578" sha1="8e8d996addd1245ca0fe853e5ee7e49e4ec1e8ad"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="othello">
+        <description>Othello (D)</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="othello.img" size="286720" crc="79c904c0" sha1="66b4267c0c79b194cf81272a538354cd2127b63b"/>
+            </dataarea>
+        </part>
+    </software>
```

Comment:
Please use parent/clone relationships, make the English version the parent, prefer to make clone short name by appending a suffix to the parent short name, and don’t unnecessarily use abbreviations in descriptions.

Reply:
Actually the German version is the original and the English version is a more recent translation of the game. This brings up a more general question if these modern image creations are even suitable to be integrated into the software list? Would it be acceptable if those were marked as clones and additionally commented as e.g. a custom translation, or should those rather not be in the list since they are not "originals"? Given that we already have custom user disks in here, I find it difficult to gauge what is accepatble.

Reply:
Added relationships and updated descriptions